config/manifests/patches/spec-descriptors.yaml

```diff
@@ -0,0 +1,452 @@
+- op: add
+  path: /spec/customresourcedefinitions/owned/0/specDescriptors
+  value:
+    - path: serverType
+      description: Server Type
+      displayName: Server Type
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:select:cassandra
+        - urn:alm:descriptor:com.tectonic.ui:select:dse
+    - path: serverVersion
+      description: Server Version
+      displayName: Server Version
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+    - path: clusterName
+      description: |
+        Name of the Cassandra cluster. Multiple CassandraDatacenter
+        objects in the same namespace and Cluster Name
+        will automatically be wired together as a single cluster.
+      displayName: Cluster Name
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+    - path: size
+      description: Size
+      displayName: Size
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+    - path: racks
+      description: "Collection of logical rack identifiers, these may be named \nand specify a zone to limit pod deployment.\n"
```

Comment:
I'm noticing a number of the `description` fields in this file are single line with `\n` characters. Is there any reason why we don't use the multi-line option like line `16` in this file?

Reply:
No idea. This is from your PR, I don't know where these came from originally. I can modify them.